client/verta/tests/versioning/environment/conftest.py

```diff
@@ -35,7 +35,6 @@ def requirements_file_with_unsupported_lines():
             "-c some_constraints.txt",
             "-f file://dummy",
             "-i https://pypi.org/simple",
-            "-e git+ssh://git@github.com/VertaAI/modeldb.git@master#egg=verta&subdirectory=client/verta",
```

Comment:
Removing this because it's no longer an "unsupported line" (instead, it gets swapped out with `"verta"` before supported-ness is even checked).